LithoUtils/Classes/Base/Models/CodableUILabelStyle.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CodableUILabelStyle.swift
+//  LithoUtils
+//
+//  Created by Remmington Damper on 1/12/22.
+//
+
+import Foundation
+import UIKit
+import Prelude
+import LithoOperators
+
+open class CodableUILabelStyle : CodableViewStyle {
+    open var labelText: String?
+    open var labelFont: CodableFont?
+    open var labelTextColor: String?
+    open var labelShadowColor: String?
```

Comment:
Let's remove the namespacing from these variables, eg `labelTextColor` should just be `textColor`, unless it directly conflicts with a superclass variable name (for instance, for UILabel.shadowColor, that's referring to the text's shadow color, not the view's shadow color, so it should probably be textShadowColor instead of just shadowColor since CodableViewStyle already has a variable named that).

Reply:
The namespacing is based on the method or property in the documentation. Do you want me to entirely rename it? That's why I have label in front so it isn't confusing.